docker/test/fasttest/run.sh

```diff
@@ -187,6 +187,8 @@ function clone_submodules
             contrib/fast_float
             contrib/NuRaft
             contrib/azure
+            contrib/boringssl
+            contrib/curl
```

Comment:
oh, well it is better not to enable building extra contribs for fast test. Better to do it the same as for other libraries -- check in `cmake/find/block_storage/cmake` contains for example file `${ClickHouse_SOURCE_DIR}/contrib/azure/cmake-modules/AzureTransportAdapters.cmake` and if not -- `set(USE_AZURE_BLOB_STORAGE 0)`.
Sorry did not take a look at the reason fast test failed before now.